brainframe/api/stubs/zone_statuses.py

```diff
@@ -66,15 +68,15 @@ def zone_status_iterator():
                     raise bf_errors.ServerNotReadyError(message)
 
                 if packet == b'':
-                    continue
-
-                # Parse the line
-                zone_statuses_dict = json.loads(packet)
-
-                processed = {
-                    int(s_id): {key: bf_codecs.ZoneStatus.from_dict(val)
-                                for key, val in statuses.items()}
-                    for s_id, statuses in zone_statuses_dict.items()}
+                    processed={}
```

Comment:
I don't see the point in yielding an empty byte. Please return the to the previous implementation (using `continue`), unless there was a specific reason to do this.

Reply:
This change fixed the issue where the client is stuck there and never returns if a stream is gone before the client exits.